src/components/Posts/PostMenu.tsx

```diff
@@ -101,6 +109,11 @@ const PostMenu = ({
         postId={postId}
         isExpandedPost={isExpandedPost}
       />
+      <EditPostModal
+        onClose={() => setEditModalOpen(false)}
+        open={editModalOpen}
+        postId={postId}
+      ></EditPostModal>
```

Comment:
Let's self close this

```suggestion
      <EditPostModal
        onClose={() => setEditModalOpen(false)}
        open={editModalOpen}
        postId={postId}
      />
```

Reply:
Done